src/components/Product/ProductBox.vue

```diff
@@ -2,7 +2,7 @@
   <div class="product-box">
       <div class="row">
         <div class="col-4" @click="showDetails">
-            <img class="img-fluid" v-bind:src="product.imageURL" alt="product-image">
+            <img class="img-fluid" v-bind:src="this.imgUrl" alt="product-image">
```

Comment:
why are using this?

Reply:
here too same
data(){
        return{
            imgUrl:''
        }
    },
    mounted(){
        this.imgUrl = this.product.imageURL
    }